### ISSUE TITLE
curl image bug: "rustls: failed to load trusted certificates"

### ISSUE THREAD

**Issue by jdolitsky** [chainguard-images/images#302]
### Which image/versions are related to this issue/feature request?

curl

### Issue/Feature description

```
$ docker run --rm cgr.dev/chainguard/curl:latest -v https://raw.githubusercontent.com/chainguard-images/images/main/images/curl/configs/latest.apko.yaml
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying x.x.x.x:443...
* Connected to raw.githubusercontent.com (x.x.x.x) port 443 (#0)
* ALPN: offers h2,http/1.1
* rustls: failed to load trusted certificates
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
* Closing connection 0
curl: (77) rustls: failed to load trusted certificates
```

**Comment by jdolitsky** [comment]
confirmed not related to newer apko etc.

this image from 2 days ago fails the same:
```
export IMAGE_NAME=cgr.dev/chainguard/curl@sha256:ffe2e14b77c2fe631a24729ee266f07393795278f32615865b024f6ce7ec8f99
./tests/02-fetch-url.sh 
```

**Comment by dlorenc** [comment]
We don't install ca-certs in the image, is it that?

**Comment by jdolitsky** [comment]
updated https://github.com/chainguard-images/images/pull/303 to add it

**Comment by jdolitsky** [comment]
resolved in #303
